appendix/contributing.md

```diff
@@ -0,0 +1,29 @@
+Contributing
+---------------------------
+
+Prerequisites
+---------------------------
+
+Reviewing the following will prepare you for contributing
+
+* You need to be comfortable with git, GitHub workflow of fork, branch, commit, open pull request, reviewing changes, and merge to work effectively in the KubeVirt community.
+* You need to be familiar with writing code in golang.  See the [golang tour](https://tour.golang.org/welcome/1) to familiarize yourself.
+* Familiarize yourself with the various repositories of the [KubeVirt](https://github.com/kubevirt) GitHub organization.
+* Read the Developer [contribution page](https://github.com/kubevirt/kubevirt/blob/master/CONTRIBUTING.md) and the [getting started page](https://github.com/kubevirt/kubevirt/blob/master/docs/getting-started.md).
+* Try the Deploy KubeVirt on [minikube](https://kubevirt.io/quickstart_minikube/) or [kind](https://kubevirt.io/quickstart_kind/) quickstart lab.
+* Try the [Using KubeVirt](https://kubevirt.io/labs/kubernetes/lab1.html) lab.
+
+Your first contribution
+---------------------------
+
+The following will help you decide where to start
+
+* Check a repository issues list for issues that make good entry points.
+* Open a pull request using GitHub to existing documentation.
```

Comment:
Maybe it's just me, but I don't really understand this sentence... "to existing documentation"?